resources/addresses/models/address_writable.yml

```diff
@@ -3,6 +3,12 @@ type: object
 required:
   - address_line1
 
+oneOf:
```

Comment:
This should be `anyOf`. "Either name or company is required, you may also add both." https://docs.lob.com/#addresses_create